pkg/sampling/internal/bytes/threshold.go

```diff
@@ -0,0 +1,101 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bytes
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"strconv"
+	"strings"
+)
+
+const (
+	// MaxAdjustedCount is 2^56 i.e. 0x100000000000000 i.e., 1<<56.
+	MaxAdjustedCount = 1 << 56
+
+	// NumHexDigits is the number of hex digits equalling 56 bits.
+	NumHexDigits = 56 / 4
+
+	hexBase = 16
+)
+
+// Threshold used to compare with the least-significant 7 bytes of the TraceID.
+type Threshold struct {
+	bytes [8]byte
+}
+
+var (
+	// ErrTValueSize is returned for t-values longer than NumHexDigits hex digits.
+	ErrTValueSize = errors.New("t-value exceeds 14 hex digits")
+
+	NeverSampleThreshold = Threshold{bytes: [8]byte{
+		0, 0, 0, 0, 0, 0, 0, 0,
+	}}
+	AlwaysSampleThreshold = Threshold{bytes: [8]byte{
+		1, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
It seems the 'bytes' for NeverSampleThreshold and AlwaysSampleThreshold are never referenced. If this is true, there's no need for keeping 8 bytes. Am I missing something?